app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Log;
+
+class PollTwitter extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'suggestive:poll-twitter';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Polls Twitter for any tweets with the configured hashtag.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $users = User::where('role', 'owner')
+                     ->whereNotNull('hashtag')
+                     ->get([ 'id', 'name', 'hashtag', 'last_tweet_id', ]);
+
+        if ($users) {
+            $twitter = app('TwitterClient');
+
+            Log::info('Found owners to poll Twitter for', [ 'count' => $users->count(), ]);
+
+            $users->each(function ($user) use ($twitter) {
+                Log::info('Polling Twitter', [ 'owner' => $user->name, 'hashtag' => $user->hashtag, ]);
+
+                $response = $twitter->get('search/tweets', [ 'q' => $user->hashtag, 'since_id' => $user->last_tweet_id, ]);
```

Comment:
Sadly [not](https://dev.twitter.com/rest/reference/get/search/tweets). Twitter lets you search for tweets `until` a given date, or since a given id, handling the filtering on their end, based on the query. This is why at the end of the process we store the first (most recent) returned response to search after.

Also, the API will return only `<limit>` (currently 100) results at a time, so what you specify as the `since_id` will be forced to the oldest available.